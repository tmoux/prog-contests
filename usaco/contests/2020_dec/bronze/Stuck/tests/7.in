20
E 45791 5057
N 91521 96157
E 31491 1947
N 59231 35021
N 93740 85054
E 54098 35325
N 87516 53516
E 72231 46139
E 85404 82338
E 19218 39021
E 39862 84812
N 94977 92685
E 99904 24176
N 79207 99759
N 89744 93499
N 20127 63950
E 77560 67818
N 10563 40049
E 88711 41805
E 63291 77375
