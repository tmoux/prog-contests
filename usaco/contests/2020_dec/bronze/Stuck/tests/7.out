Infinity
Infinity
Infinity
49791
Infinity
5133
14302
Infinity
Infinity
40013
Infinity
Infinity
Infinity
Infinity
Infinity
Infinity
Infinity
Infinity
Infinity
Infinity
